src/store/metadata/stub/content/datasets.json

```diff
@@ -701,8 +701,57 @@
           "modified": "2023-06-05T16:27:00Z"
         }
       ]
+    },
+    {
+      "@id": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates",
+      "@type": "dcat:DatasetSeries",
+      "identifier": "annual-personal-wellbeing-estimates",
+      "title": "Annual personal wellbeing estimates",
+      "summary": "Estimates of life satisfaction, feeling that the things done in life are worthwhile, happiness and anxiety for adults aged 16 years and over, UK, 04/2011-03/2012 to 04/2022-03/2023.",
+      "description": "This dataset shows the average rating of those that feel satisfied about their lives (where 0 is 'not at all satisfied' and 10 is 'completely satisfied'), for adults aged 16 years and over, in the UK, for periods 04/2011-03/2012 to 04/2022-03/2023.",
+      "issued": "2023-09-13T13:32:00Z",
+      "modified": "2023-11-28T14:40:00Z",
+      "next_release": "2024-09-01T09:30:00Z",
+      "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "creator": "https://staging.idpd.uk/publishers/department-for-education",
```

Comment:
Please change to `"https://staging.idpd.uk/publishers/office-for-national-statistics"`

A good reference is the published dataset on PMD-Beta

Reply:
noted

Reply:
okay